Source/WebKit/Shared/EntryPointUtilities/Cocoa/XPCService/XPCServiceEntryPoint.h

```diff
@@ -88,48 +88,11 @@ void initializeAuxiliaryProcess(AuxiliaryProcessInitializationParameters&& param
 void setOSTransaction(OSObjectPtr<os_transaction_t>&&);
 #endif
 
+void setJSCOptions(bool configureJSCForTesting, bool enableLockdownMode, bool disableJIT, bool enableSharedArrayBuffer, bool disableJITCage);
```

Comment:
No, it's not a good design to have a function that takes many booleans like this. It's too easy to pass parameters in the wrong order and introduce bugs.

Reply:
Good point! I'll update the patch.

Thanks for reviewing!